M2/Macaulay2/d/chrono.dd

```diff
@@ -0,0 +1,78 @@
+-- TODO: get core time: number of cpu clock ticks spent in the core/libraries
+-- TODO: define time and elapsedTime in the top level instead, and print in the AfterPrint
+-- TODO: auto-calibrate spin based on the wall time
```

Comment:
Oh, I should say that I wrote this commit a long time ago. I don't really agree with the second item anymore.